src/main/resources/anniv5Resources/localization/eng/madsciencepack/Cardstrings.json

```diff
@@ -1,45 +1,45 @@
 {
   "${ModID}:Sharpen": {
     "NAME": "Honing Strike",
-    "DESCRIPTION": "Deal !D! damage. NL ${ModID}:Modify a random Attack in hand: 'Add !M! Damage'"
+    "DESCRIPTION": "Deal !D! damage. NL ${ModID}:Modify a random Attack in hand: \"Add !M! Damage\""
```

Comment:
I see that this is the solution to the odd text/keyword parsing issues you were seeing. Glad you got that figured out.